UDPClientGUI/SiceUDPClient.cs

```diff
@@ -51,5 +59,25 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ClientCommandEvent Command)
+        {
+            ConnectionIP = Command.ConnectionIP;
+            ConnectionPort = Command.ConnectionPort;
+            if (Command.ClientCommand == ClientCommandEvent.Command.Connect) {
+                Connect();
+                return;
+            }
+            if (Command.ClientCommand == ClientCommandEvent.Command.Disconnect) {
+                Disconnect();
+                return;
+            }
+            if (Command.ClientCommand == ClientCommandEvent.Command.SendMessage)
+            {
+                InputMessage = Command.Message;
+                SendMessage();
+                return;
+            }
```

Comment:
As an exercise, try to replace these with Action

You can define a dictionary of key: ClientCommand and value: action

Reply:
Used dictionary